bin/xbundle_convert.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+"""
+A command-line script to use xbundle.
+"""
+
+import sys
+
+from xbundle import XBundle, RunTests
+
+# pylint: disable=star-args
+
+def usage():
```

Comment:
totally out of scope for this change, but we really should switch to using argparse